timemachine/fe/absolute_hydration.py

```diff
@@ -351,4 +351,4 @@ def run_solvent(
         n_windows=n_windows,
         steps_per_frame=steps_per_frame,
     )
-    return solvent_res, solvent_top
+    return solvent_res, solvent_top, solvent_host_config
```

Comment:
I think https://github.com/proteneer/timemachine/blob/73ea83a18de73b5bd6a2836ef6828d4eb20082d8/tests/test_fe_absolute_hydration.py#L16 needs to be updated to match